runtime/src/prioritization_fee_cache.rs

```diff
@@ -197,10 +197,14 @@ impl PrioritizationFeeCache {
     pub fn update<'a, Tx: TransactionWithMeta + 'a>(
         &self,
         bank: &Bank,
-        txs: impl Iterator<Item = &'a Tx>,
+        txs: impl Iterator<Item = Option<&'a Tx>>,
```

Comment:
why not just filter them out before this? i.e. use `filter_map` on the iterator instead of `map` to an option?

Reply:
Yeah, your suggestion is way better; I'll do that